drizzlepac/haputils/catalog_utils.py

```diff
@@ -690,17 +690,18 @@ def verify_crthresh(self, n1_exposure_time):
         log.info("  based on EXPTIME = {}sec for the n=1 filters".format(n1_exposure_time))
 
         detector = self.image.keyword_dict["detector"].upper()
-        for cat_type in self.catalogs:
-            source_cat = self.catalogs[cat_type]
-            if source_cat.sources:
-                thresh = self.crfactor[detector][cat_type] * n1_exposure_time**2 / self.image.keyword_dict['texpo_time']
-                source_cat = source_cat.sources if cat_type == 'aperture' else source_cat.source_cat
-                n_sources = source_cat.sources_num_good  # len(source_cat)
-                all_sources = len(source_cat)
-                log.info("{} catalog with {} good sources out of {} total sources :  CR threshold = {}".format(cat_type, n_sources, all_sources, thresh))
-                if n_sources < thresh and 0 < n_sources:
-                    self.reject_cats[cat_type] = True
-                    log.info("{} catalog FAILED CR threshold.".format(cat_type))
+        if detector not in ['IR', 'SBC']:
+            for cat_type in self.catalogs:
+                source_cat = self.catalogs[cat_type]
+                if source_cat.sources:
+                    thresh = self.crfactor[detector][cat_type] * n1_exposure_time**2 / self.image.keyword_dict['texpo_time']
+                    source_cat = source_cat.sources if cat_type == 'aperture' else source_cat.source_cat
+                    n_sources = source_cat.sources_num_good  # len(source_cat)
+                    all_sources = len(source_cat)
+                    log.info("{} catalog with {} good sources out of {} total sources :  CR threshold = {}".format(cat_type, n_sources, all_sources, thresh))
+                    if n_sources < thresh and 0 < n_sources:
```

Comment:
```suggestion
                    if 0 < n_sources < thresh:
```

Reply:
At the request of DSB, I am only making code changes which absolutely need to be done for the release candidates at this time.  Please know I have a working branch with the code changes you have suggested here, as well as some changes suggested in PR#1853 which will be checked in to main after we have finished with this build.

Reply:
Never mind.  I have the OK.

Reply:
DONE